internal/cli/artifact_list.go

```diff
@@ -62,6 +62,9 @@ func (c *ArtifactListCommand) Run(args []string) int {
 		}
 		sort.Sort(serversort.ArtifactStartDesc(resp.Artifacts))
 
+		// Output the app's name before the json or table data
+		app.UI.Output("%s", app.Ref().Application, terminal.WithHeaderStyle())
```

Comment:
This will make the output not proper json though... so probably just do this in the non-json case.